check/controls.go

```diff
@@ -185,6 +200,69 @@ func (controls *Controls) JUnit() ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// AASF encodes the results of last run to AWS Security Finding Format(AASF).
```

Comment:
```suggestion
// ASFF encodes the results of last run to AWS Security Finding Format(ASFF).
```

Reply:
Prestty sure it should be ASFF, right? (https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html)

Reply:
Yes. Agree that we should update to ASFF